etc/testing/passing_test_regex.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# This script prints a regex (which should be passed to 'go test -run=<regex>')
+# that matches all tests in the pachyderm test suite that aren't in
+# etc/testing/blacklist.txt. Tests in blacklist.txt are flaky or otherwise
+# cause our test suite to consitently flake or fail when run in a cloud
+# provider's cluster and would render our nightly tests useless.
+
+basedir="$(realpath "$(dirname "${0}")/../..")"
+tmpfile="$(mktemp --dry-run --tmpdir=.)"
+
+# Put blacklisted tests in ${tmpfile}
+grep -v '^$' "${basedir}/etc/testing/blacklist.txt" >"${tmpfile}"
+
+# List all tests in src/server, and put them in ${tmpfile} as well
+ls "${basedir}/src/server" \
+  | grep '_test\.go$' \
+  | while read f; do
+      grep --no-filename '^func Test[A-Za-z_]\+(.* \*testing\.T)' "${basedir}/src/server/${f}" \
```

Comment:
That is the intent of this PR (and is currently all that `make run-bench` runs). Long-term, it might well make sense to run more, but I think sticking with `src/server` is useful for now, since I want to get the nightly build to be useful asap